app/services/etherscan_service.rb

```diff
@@ -13,10 +13,30 @@ def is_proof_member?(address)
       address: address
     )
 
-    if response.dig('result')&.last&.dig('to')&.downcase == address.downcase
+    transactions = response.dig('result')
+
+    inCount = 0
+    outCount = 0
+
+    transactions.each do |child|
+      toAddress = child.dig('to').downcase
```

Comment:
don't think the dig get's you anything here, `dig` just let's you "dig" into an hash multiple layers and if any are `nil` it will return `nil` instead of having to explicitly check for `nil` before going deeper... https://tiagoamaro.com.br/2016/08/27/ruby-2-3-dig/

I actually only used it in the above example to be able to add a safe navigation operator `&` (https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) and it not error. You can't do `hash['key']&.downcase` it will error.

In this case and bellow you can just do `transaction['to'].downcase` and `transaction['from'].downcase` given you take my suggestion of using `transaction` as the iterator instead of `child`